otelopscol/build.ps1

```diff
@@ -29,10 +29,21 @@ param (
     [string]$outDir = "."
 )
 
+# Store PATH
```

Comment:
Looks like this was edited manually. This will need to be properly generated by distrogen before merge.

You'll need to make this updates to `templates/otelopscol/build.ps1.go.tmpl`. Once you regenerate you should have the same as what you have here.

Reply:
Sounds good! I updated `build.ps1.go.tmpl` and ran distrogen. Done!